core/config.js

```diff
@@ -14,10 +20,14 @@ const variables = {
         },
         storage: {
             engine: process.env.POINT_STORAGE_ENGINE || undefined,
+            arweave_key,
```

Comment:
We don’t store keys in the config anymore. It is not safe. To get the `arweave` keys use `wallet/index.js` module instead. Also please mind that importing private keys should and will be restricted.

Reply:
Ok. I read out the wallet/index.js code and don't seem this module is handling arweave keys yet. Is that correct or I missed something? In case, if this is not implemented yet, I think maybe creating a new class to handle arweave keys should be a good idea since current wallet has a lot of code to deal with solana and other specific code.

Reply:
@diogosmendonca sorry, my bad. This functionality is currently in progress. As you can see the following methods are already available in the `wallet` module:

* `getNetworkAccount`
* `getNetworkAccountPublicKey`
* `getNetworkAccountPrivateKey`

Therefore the best solution would be to implement the below methods using the same approach.
* `getArweaveAccount`
* `getArweaveAccountPublicKey`
* `getArweaveAccountPrivateKey`

The idea is to have only one module responsible for managing access to these keys. This is just a part of a bigger plan on improving the overall app security.